### PR TITLE
Fix the miri submodule

### PR DESCRIPTION
**Pull request by oli-obk**
cc @bjorn3 

r? @eddyb

### CODE REVIEW

File: src/librustc_mir/interpret/memory.rs [hunk 1]
```diff
@@ -844,7 +844,7 @@ impl<'a, 'mir, 'tcx, M: Machine<'mir, 'tcx>> Memory<'a, 'mir, 'tcx, M> {
         }
 
         // Forget all the relocations.
-        alloc.relocations.remove_range(first ..= last);
+        alloc.relocations.remove_range(first .. last);
```
Comment (eddyb) [inline src/librustc_mir/interpret/memory.rs]: This shouldn't have spaces around `..`.

**Comment by eddyb** [comment]
r=me with nit fixed

**Comment by oli-obk** [comment]
Spurious?

```
[01:31:13] test collections::hash::map::test_map::test_lots_of_insertions ... ok
[01:31:44] test process::tests::test_process_output_fail_to_start ... test process::tests::test_process_output_fail_to_start has been running for over 60 seconds


No output has been received in the last 30m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
```

@bors retry

**Pull request merged**